web/src/main/java/se/inera/intyg/webcert/web/logging/HashPatientIdHelper.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.logging;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.regex.Pattern;
+
+public class HashPatientIdHelper {
+
+    private HashPatientIdHelper() {
+
+    }
+
+    public static String fromUrl(String url) {
```

Comment:
We have a HashUtility-class defined in schemas that we used for other scenarios when we needed to hash personalId. Should we extend that class with the function ```fromUrl``` instead of creating a new one?

Reply:
Do you mean in the schemas repo or is it a directory in WC? I asked in utv chat and got the answer to not use common repos

Reply:
Okay, I missed that conversation, sorry. A suggestion to reduce the dependency on schemas—at least partly—is to move the hash method into our new helper.

What do you think about that?